lti_consumer/plugin/urls.py

```diff
@@ -46,6 +51,11 @@
         access_token_endpoint,
         name='lti_consumer.access_token_via_location'
     ),
+    path(
+        'lti_consumer/v1/token/<str:external_app>/<slug:external_slug>',
```

Comment:
I thought the app name could be less strict, just replaced str with slug since the app name should be a slug to, just applied this fix on the latest commit.